Makefile-ponyc

```diff
@@ -190,10 +193,10 @@ define USE_CHECK
     ALL_CFLAGS += -DUSE_POOLTRACK
     PONY_BUILD_DIR := $(PONY_BUILD_DIR)-pooltrack
   else ifeq ($1,dtrace)
-    DTRACE ?= $(shell which dtrace)
-    ifeq (, $(DTRACE))
```

Comment:
Another fix would be to change this line to be:

```
    ifeq (, $$(DTRACE))
```

The extra `$` tells make to evaluate that variable when that block is `eval`d instead of when the file is loaded.

Reply:
I've force-pushed a fix using this method, thanks!